docs/docker/using-keycloak.md

```diff
@@ -56,10 +56,21 @@ documentation](../Authenticating-and-Authorizing-Users-via-keycloak.md#configure
 Remember to specify port 8180 for the Keycloak server, wherever the guide says
 8080.
 
-When starting the cBioPortal web server with the new configuration, instead of
-modifying Tomcat config files, include the `-Dauthenticate=saml` flag in the
-`JAVA_OPTS` argument on the command line:
+After configuring Keycloak, set up cBioPortal containers [as specified in the documentation](README.md). Make sure to replace
+the `JAVA_OPTS` argument in Step 6 to include the `-Dauthenticate=saml` flag:
 
-```
-    -e JAVA_OPTS='-Xms2g -Xmx4g -Dauthenticate=saml' \
-```
+<pre>
+docker run -d --restart=always \
+    --name=cbioportal-container \
+    --net=cbio-net \
+    -v /<path_to_config_file>/portal.properties:/cbioportal/portal.properties:ro \
```

Comment:
@oplantalech I think the ` /<path_to_config_file>/` needs to be escaped somehow - if you look at the rich diff it shows up as `//`

Reply:
![Screen Shot 2019-12-18 at 11 20 31 AM](https://user-images.githubusercontent.com/1334004/71103632-7034d180-2188-11ea-9638-5eb888308ff7.png)